app/src/main/kotlin/com/proxerme/app/helper/MaterialDrawerHelper.kt

```diff
@@ -133,17 +157,32 @@ class MaterialDrawerHelper(context: Activity, toolbar: Toolbar,
 
     private fun buildDrawer(context: Activity, toolbar: Toolbar, accountHeader: AccountHeader,
                             savedInstanceState: Bundle?): Drawer {
-        return DrawerBuilder(context)
+        val stickyDrawerItems = generateStickyDrawerItems()
+        this.stickyItemIds = stickyDrawerItems.mapNotNull { DrawerItem.fromOrNull(it.identifier) }
+                .toTypedArray()
```

Comment:
Init this in the constructor. Also please inline `generateStickyDrawerItems` directly into the mapping call:

```kt
this.stickyItemIds = generateStickyDrawerItems().mapNotNull { DrawerItem.fromOrNull(it.identifier) }                                          
        .toTypedArray()
```